docs/product-getting-started-guide/src/main/asciidoc/task_business-process.adoc

```diff
@@ -49,7 +47,8 @@ image::bus-rule-task.png[]
 +
 * *Name*: `Validation`.
 * *Ruleflow group*: `validation`.
-* Click image:btn_assign.png[] under *Assignments*. In the *Validation Data I/O* dialogue, create the following assignments:
+* Scoll down to the *Task Data* section and click image:btn_assign.png[] under *Assignments*.
```

Comment:
Typo: "Scroll"

Also, a strange hierarchy between step 6 and 7, in this area. the "Scroll down to" and the "In the Validation Data" points seem like they should be steps following step 6, but instead or bullets, followed by sub-bullets, as if they're associated with the preceding bullets rather than the preceding steps. If this is a nested section for step 6 above, then the steps should be a. and b., not bullets. I'd just confirm the flow and confirm what exactly is the process in this small section.

Reply:
Thank you Stetson. "Scroll" fixed. Also, I quickly reorg'd the steps that you noticed. There is no reason for the previous hierarchy.